packages/trader/src/App/Containers/Modals/trade-modals.jsx

```diff
@@ -7,9 +7,11 @@ import { observer, useStore } from '@deriv/stores';
 import { useTraderStore } from 'Stores/useTraderStores';
 
 const TradeModals = observer(() => {
-    const { ui, client, common } = useStore();
+    const { ui, client, common, traders_hub } = useStore();
     const { resetPreviousSymbol, clearPurchaseInfo, requestProposal: resetPurchase } = useTraderStore();
     const { is_virtual, is_logged_in } = client;
+    const { multipliers_account_status } = traders_hub;
```

Comment:
what do you think about moving `multipliers_account_status` to Shared package and reuse it in other places instead of creating a new connection between Trader and TradersHub?

Reply:
i moved the logic to client store since it is  client status , removed the connection that i made between Trader and Traders -hub